hal/ia32/Makefile

```diff
@@ -7,11 +7,15 @@
 
 include hal/tlb/Makefile
 
-OBJS += $(addprefix $(PREFIX_O)hal/ia32/, _init.o _exceptions.o _interrupts.o spinlock.o exceptions.o interrupts.o cpu.o pmap.o timer.o hal.o string.o pci.o init.o)
+OBJS += $(addprefix $(PREFIX_O)hal/ia32/, _init.o _exceptions.o _interrupts.o spinlock.o exceptions.o interrupts.o cpu.o pmap.o timer.o hal.o string.o pci.o init.o console.o)
 CFLAGS += -Ihal/ia32
 
-ifeq ($(CONSOLE), vga)
-	OBJS += $(PREFIX_O)hal/ia32/console-vga.o
-else
```

Comment:
`CONSOLE=serial` was the default. Now, if `CONSOLE` is not defined, there will be no output, right?

Reply:
`vga` was the default and still is thanks to the assignment in `build.project`
https://github.com/phoenix-rtos/phoenix-rtos-project/blob/master/_targets/ia32/generic/build.project#L14

Reply:
Nice